sql/core/src/main/scala/org/apache/spark/sql/columnar/InMemoryColumnarTableScan.scala

```diff
@@ -43,7 +42,7 @@ private[sql] object InMemoryRelation {
       tableName)()
 }
 
-private[sql] case class CachedBatch(buffers: Array[Array[Byte]], stats: InternalRow)
+private[sql] case class CachedBatch(count: Int, buffers: Array[Array[Byte]], stats: InternalRow)
```

Comment:
let's add classdoc for count, buffers, and stats.

Without looking at the code it is less clear what count is here. (Is it the total number of rows?)